azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/internal/AzureLiveTestUtils.java

```diff
@@ -28,6 +30,7 @@ public static Properties defaultProperties(Properties properties) {
        properties.put("oauth.credential", "password");
        properties.put("oauth.endpoint", "https://login.microsoftonline.com/oauth2/token");
        properties.put(CREDENTIAL_TYPE, CLIENT_CREDENTIALS_SECRET.toString());
+       properties.put(PROPERTY_REGIONS, "northeurope,eastus");
```

Comment:
When rolling back the location transformation function you'll probably have to change these values. They also appear in the README as an example, so bear in mind to change that too.